### DISCUSSION TITLE
createNft and minting an edition

### DISCUSSION THREAD

**Discussion by katewayo** [metaplex-foundation/js#58] [Q&A] [answered]
Hi all. If createNft is called to create a limited edition NFT mint, can the master edition returned in the output be used to create the remaining limited editions? If so, via what API? I don't see a way to pass the master edition to createNft. Thanks in advance for the help!

Take care,

enderzshadow.sol 
Twitter: @katewayo

**Comment by lorisleiva** [comment, marked as answer]
Hi 👋 That's a good question, the NFT module is still under development and we're planning on adding more features such as minting a limited edition from a master edition. This is actually quite high on my priority list so it should be released fairly soon but, at this time, this is not doable with the new SDK.

**EDIT 2022-04-22:** This is now supported from version `0.7.0`.

**Reply by katewayo** [reply]
Thank you! Sounds like I can rely on a future mechanism being available that uses the master edition id to mint additional limited editions. That works well with my current implementation design. I'll stay tuned for more SDK updates.

**Reply by lorisleiva** [reply]
Hi 👋 Just to let you know, we've released this feature in version `0.7.0`.

📚 [Here's a link to the documentation](https://github.com/metaplex-foundation/js-next#printNewEdition).
